scripts/enforceVersions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+if [[ $(pwd) =~ .+simorgh ]]; then
+  desired_node_version=$(cat .nvmrc)
+
+  if [[ $(node -v) != "$desired_node_version"* ]]; then
+    tput setaf 1
+    echo "Oh no! You're not using our version of node, please move to $desired_node_version"
+    tput sgr0
```

Comment:
```suggestion
    tput setaf 3
```
Set to yellow to make 2nd line stand out in the output